sources/packages/backend/libs/services/src/health-check/zeebe-health-indicator.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@nestjs/common";
+import { Workers } from "@sims/services/constants";
+
+@Injectable()
+export class ZeebeHealthIndicator {
+  private readonly workersConnectionStatus: Record<string, boolean> = {};
+
+  reportConnectionWorkerStatus(workerName: string, status: boolean) {
+    this.workersConnectionStatus[workerName] = status;
+  }
+
+  allConnected(): boolean {
+    const statuses = Object.values(this.workersConnectionStatus);
+    if (statuses.length !== Object.keys(Workers).length) {
```

Comment:
I would say `Object.keys(Workers).length` can be a constant something like`totalWorkerCount` for better readability